custom-vu/src/main/kotlin/jces1209/vu/page/admin/ViewProjects/CloudViewProjectsPage.kt

```diff
@@ -0,0 +1,25 @@
+package jces1209.vu.page.admin.ViewProjects
```

Comment:
Let's use the unified format:
1. make lowercase and general name for package `jces1209.vu.page.admin.projects`
2. use Browse for the list and View for single entity - `BrowseProjects`

Reply:
Let's rename it as Manage projects

![image](https://user-images.githubusercontent.com/65538610/90504775-4211f200-e15a-11ea-93e3-c406e9d09954.png)

Reply:
And we already have BrowseProjects class

Reply:
And we already have BrowseProjects class